plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+--------------------------------------------------------------------------------
+Prepare backup Directory: 
+Creating backup folder /tmp/some/dir/katello-backup-2018-05-03-03-20-35
+                                                                      [OK]
+--------------------------------------------------------------------------------
+Check if the directory exists and is writable:                        [OK]
+--------------------------------------------------------------------------------
+Generate metadata:                                                    [SKIPPED]
+--------------------------------------------------------------------------------
+Backup config files: 
+\ Collecting config files to backup                                   [OK]      
+--------------------------------------------------------------------------------
+Backup Pulp data: 
+- Collecting Pulp data                                                [OK]      
+--------------------------------------------------------------------------------
+Backup Mongo online: 
+| Getting dump of Mongo DB                                            [OK]      
+--------------------------------------------------------------------------------
+Backup Postgres global objects online:                                [OK]
+--------------------------------------------------------------------------------
+Backup Candlepin database online: 
+\ Getting Candlepin DB dump                                           [OK]      
+--------------------------------------------------------------------------------
+Backup Foreman database online: 
+- Getting Foreman DB dump                                             [OK]      
+--------------------------------------------------------------------------------
+Compress backup data to save space:                                   [OK]
+--------------------------------------------------------------------------------
+
+Done with backup: 2018-05-03 03:20:42 +0000
+**** BACKUP Complete, contents can be found in: /tmp/some/dir/katello-backup-2018-05-03-03-20-35 ****
+```
+You can see the metadata step was skipped.
+
+To see a list of available step labels use:
+```
+foreman-maintain advanced procedure run -h
+``` 
+
+In some cases it make sense to run one individual step, e.g. collect config files for debugging purposes:
+```
+# foreman-maintain advanced procedure run backup-config-files --backup-dir /tmp/backup
+Running ForemanMaintain::Scenario
+================================================================================
+Backup config files: 
+\ Collecting config files to backup                                   [OK]      
+--------------------------------------------------------------------------------
+```
+ 
+### Directory names
+By default the backup is stored in subdirectory named `<katello|foreman|satellite|capsule>-backup-YYYY-MM-DD-hh-mm-ss` in a directory provided on a command line.
```

Comment:
` in a subdirectory`

Reply:
`<katello|foreman|satellite|foreman-proxy|capsule>-backup-YYYY-MM-DD-hh-mm-ss`